Formula/tctl.rb

```diff
@@ -15,7 +15,9 @@ class Tctl < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args, "-ldflags", "-s -w", "./cmd/tools/cli/main.go"
+    system "go", "build", *std_go_args, "-ldflags", "-s -w", "-o", bin/"tctl", "./cmd/tools/cli/main.go"
+    system "go", "build", *std_go_args, "-ldflags", "-s -w", "-o", bin/"tctl-authorization-plugin",
+      "./cmd/tools/cli/plugins/authorization/main.go"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags:"-s -w"), "./cmd/tools/cli/main.go"
    system "go", "build", *std_go_args(ldflags:"-s -w"), "-o", bin/"tctl-authorization-plugin", "./cmd/tools/cli/plugins/authorization/main.go"
```

Reply:
`brew lint` complains the line was too long unless I split it.

Reply:
Updated and re-split line. And I meant `brew audit`.